### PR TITLE
Publish telltime.0.0.1

### PR DESCRIPTION
**Pull request by darrenldl**
Cli tool for interacting with Daypack-lib components

Homepage: https://github.com/daypack-dev/telltime
Repo: git+https://gitlab.com/daypack-dev/telltime.git
Issue tracker: https://github.com/daypack-dev/telltime/issues

### CODE REVIEW

File: packages/telltime/telltime.0.0.1/opam [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+"""
+maintainer: "Darren Ldl <darrenldldev@gmail.com>"
+authors: "Daypack developers"
+license: "MIT"
+homepage: "https://github.com/daypack-dev/telltime"
+bug-reports: "https://github.com/daypack-dev/telltime/issues"
+dev-repo: "git+https://github.com/daypack-dev/telltime"
+depends: [
+  "ocaml" {>= "4.08.0"}
+  "dune" {>= "1.11.0"}
```
Comment (mseri) [inline packages/telltime/telltime.0.0.1/opam:14]:
```suggestion
  "dune" {>= "2.6"}
```

Your dune-project file requires dune 2.6

Comment (mseri) [reply packages/telltime/telltime.0.0.1/opam]: Can you upstream the change?

**Comment by mseri** [comment]
Feel free to announce the release on the discuss.ocaml.org forum. We have a community caterogy and an announcement tag for this purpose

File: packages/telltime/telltime.0.0.1/opam (hunk 1 above)
Comment (darrenldl) [reply packages/telltime/telltime.0.0.1/opam]: Aha, didn't realise this project used `lang dune 2.6`. Thanks and done.

**Pull request merged**